observation_portal/proposals/models.py

```diff
@@ -155,16 +158,20 @@ def __str__(self):
         return self.id
 
     def as_dict(self):
-        proposal = model_to_dict(self, exclude=[])
+        proposal = model_to_dict(self, exclude=['notes', 'users'])
```

Comment:
Excluding `notes` because that field is meant for admins only. It gets edited on the admin site, and also shows up on the semester admin table, so it is not needed here.